.github/workflows/test_bwc.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       # anomaly-detection
       - name: Checkout AD
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

Comment:
why downgrade to checkoutv3?

Reply:
mentioned the reason in the Description:

* checkout failed due to missing library
  * error message: Run actions/checkout@v4
/usr/bin/docker exec  cdc91410f06d7ab184b358bd420df75fe15958c0071a4fb949ff7724f0ecdcbf sh -c "cat /etc/*release | grep ^ID"
/__e/node20/bin/node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /__e/node20/bin/node)
/__e/node20/bin/node: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /__e/node20/bin/node)
/__e/node20/bin/node: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /__e/node20/bin/node)
/__e/node20/bin/node: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /__e/node20/bin/node)
/__e/node20/bin/node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /__e/node20/bin/node)
/__e/node20/bin/node: /lib64/libc.so.6: version `GLIBC_2.25' not found (required by /__e/node20/bin/node)
  * fix: reverted: https://github.com/opensearch-project/anomaly-detection/pull/1098/files

Reply:
sorry missed this